README.md

```diff
@@ -12,50 +12,54 @@ Given the recent (and developing) situation with decision in the .net core effor
 # Examples
 ### Finding a Store
 ```C#
+var api = new DominosApi();
 var deliveryAddress = new Address("9 Lupine Road", "Lowell", "MA", "01850", Address.UnitCategory.House);
-var locationData = await SearchLocations(deliveryAddress);
+var locationData = await api.SearchLocations(deliveryAddress);
 
 int? nearestStoreID = locationData.Stores.OrderBy(x => x.MinDistance)
     .FirstOrDefault(x => x.ServiceIsOpen?.Delivery.Value ?? false)?.StoreID;
 ```
 
 ### Searching the Menu
 ```C#
-var menu = await GetMenu(nearestStoreID.Value);
+var api = new DominosApi();
+var menu = await api.GetMenu(nearestStoreID.Value);
 var practicalMenuInfo = menu.PreconfiguredProducts.Select(x =>
     new Tuple<string, string, string>(x.Value.Code, x.Value.Name, x.Value.Description));
 ```
 
 ### Placing an Order
 ```C#
+var api = new DominosApi();
 var deliveryAddress = new Address("43 Elm Avenue", "Quincy", "MA", "02170", Address.UnitCategory.House);
 var customerInfo = new Customer("John", "Cheever", deliveryAddress, "thecheeves@gmail.com", "6173456789");
 string storeId = "3712";    // Get this value from a call to SearchLocations(...)
 
 // Two medium pepperoni pizzas, one medium cheese, and cinnastix
 var products = new List<Product> {  new Product("14SCREEN", 2, new List<string> { "C", "P", "X" }),
-                                    new Product("14SCREEN", 1, new List<string> { "C", "X" }),
-                                    new Product("CINNASTIX8", 1)};
+	new Product("14SCREEN", 1, new List<string> { "C", "X" }),
+	new Product("CINNASTIX8", 1)};
 
 var coupons = new List<Coupon> { new Coupon("3510", 1) };   // This particular coupon saves us money on three medium pizzas.
-                                                            // Get this value from a call to GetMenu(...)
+// Get this value from a call to GetMenu(...)
```

Comment:
I think this looks a little clearer (at least in the github readme interface) when these comments are lined up.